pdbtools/pdb_tidy.py

```diff
@@ -143,16 +143,23 @@ def make_TER(prev_line):
     fmt_TER = "TER   {:>5d}      {:3s} {:1s}{:>4s}{:1s}" + " " * 53 + "\n"
 
     records = ('ATOM', 'HETATM')
-    ignored = ('TER', 'END ', 'END\n', 'CONECT', 'MASTER')
+    ignored = ('TER', 'END', 'CONECT', 'MASTER', 'ENDMDL')
     # Iterate up to the first ATOM/HETATM line
     prev_line = None
+    num_models = 1
+    in_model = False
     for line in fhandle:
 
+        line = line.strip()  # We will pad/add \n later to make uniform
```

Comment:
@amjjbonvin  The new `strip` here before the `if statements` avoids the need to repeat `END` in the `ignore` and also catches some `END` cases not caught before. For example, in the previous version running `pdb_tidy` in `tests/data/ensemble_error_2.pdb` was adding **two** `END` lines:

```
TER       3      ASN A   1                                                      
ENDMDL                                                                          
END                                                                             
END
```

Now this behaviour is corrected.